tests/components/group/test_init.py

```diff
@@ -652,6 +652,13 @@ async def test_is_on(hass: HomeAssistant) -> None:
             (STATE_ON, True),
             (STATE_OFF, False),
         ),
+        (
+            ("cover", "lock"),
+            (STATE_OPEN, STATE_OPEN),
+            (STATE_CLOSED, STATE_LOCKED),
+            (STATE_ON, True),
+            (STATE_OFF, False),
+        ),
```

Comment:
```suggestion
        (
            ("cover", "lock"),
            (STATE_OPEN, STATE_OPEN),
            (STATE_CLOSED, STATE_LOCKED),
            (STATE_ON, True),
            (STATE_OFF, False),
        ),
```

Test is not part of the rework.